src/make_xlsx.py

```diff
@@ -544,6 +545,48 @@ def transform(x):
             for h, v in line.items():
                 worksheet.write(i, refs[h][0], refs[h][2](v), refs[h][3])
 
+def gen_s3_cost(workbook, header_format, val_format):
+    def transform(x):
+        try:
+            if x == "": return 0.0
+            else: return float(x)
+        except ValueError:
+            return x
+    with open(IN_S3_COST) as f:
+        reader = csv.DictReader(f)
+        worksheet = workbook.add_worksheet("S3 cost")
+
+        worksheet.merge_range("A1:F1", "S3 cost for current month", header_format)
+
+        cur_format = workbook.add_format()
+        cur_format.set_align("center")
+        cur_format.set_align("vcenter")
+        cur_format.set_border()
+        cur_format.set_num_format(NUMFORMAT_CURRENCY)
+
+        worksheet.set_column(0, 0, 45)
```

Comment:
It would be better to freeze the header. You can do it with `worksheet.freeze_panes`